changelogs/fragments/254-add_ignore_txt.yml

```
@@ -0,0 +1,2 @@
+trivial:
```

```suggestion
---
trivial:
```

The linter requires the start of document directive to be present

---

@Qalthos committed, thanks